Fix | Exclude unsupported protocols (#1168)
dotnet/SqlClient: BUILDGUIDE.md

**"Switch.Microsoft.Data.SqlClient.LegacyRowVersionNullBehavior"**

## Enabling OS secure protocols preference

> Tls 1.3 has been taken out in view of the fact that the driver lacks full support. To jump back on the OS preferences as same as earlier, you must enable the following AppContext switch on application's startup:

Reviewer: TLS ?

Reviewer 2: Yes, ditto.
```suggestion
TLS 1.3 has been excluded due to the fact that the driver lacks full support. To enable OS preferences as before, enable the following AppContext switch on application startup:
```

> **"Switch.Microsoft.Data.SqlClient.EnableSecureProtocolsByOS"**

Reviewer 2: I would name the switch `Switch.Microsoft.Data.SqlClient.UseSystemDefaultSecureProtocols`.
